src/Components/Facility/Consultations/DailyRoundsList.tsx

```diff
@@ -6,88 +6,99 @@ import { useTranslation } from "react-i18next";
 import LoadingLogUpdateCard from "./DailyRounds/LoadingCard";
 import routes from "../../../Redux/api";
 import PaginatedList from "../../../CAREUI/misc/PaginatedList";
+import PageTitle from "../../Common/PageTitle";
+import { useState } from "react";
+
+import { DailyRoundsFilter } from "./DailyRoundsFilter";
 
 export const DailyRoundsList = (props: any) => {
   const { t } = useTranslation();
-  const {
-    facilityId,
-    patientId,
-    consultationId,
-    consultationData,
-    showAutomatedRounds,
-  } = props;
+  const { facilityId, patientId, consultationId, consultationData } = props;
+
+  const [filter, setFilter] = useState({
+    roundType: "",
+    fromDateTime: "",
+    toDateTime: "",
+  });
 
   return (
-    <PaginatedList
-      route={routes.getDailyReports}
-      pathParams={{
-        consultationId,
-      }}
-      query={{
-        rounds_type: showAutomatedRounds ? "" : "NORMAL,VENTILATOR,ICU",
-      }}
-    >
-      {(_) => (
-        <div className="-mt-2 flex w-full flex-col gap-4">
-          <div className="flex max-h-[85vh] flex-col gap-4 overflow-y-auto overflow-x-hidden px-3">
-            <PaginatedList.WhenEmpty className="flex w-full justify-center border-b border-gray-200 bg-white p-5 text-center text-2xl font-bold text-gray-500">
-              <span className="flex justify-center rounded-lg bg-white p-3 text-gray-700 shadow">
-                {t("no_consultation_updates")}
-              </span>
-            </PaginatedList.WhenEmpty>
-            <PaginatedList.WhenLoading>
-              <>
-                {Array.from({ length: 3 }).map((_, i) => (
-                  <LoadingLogUpdateCard key={i} />
-                ))}
-              </>
-            </PaginatedList.WhenLoading>
-            <PaginatedList.Items<DailyRoundsModel> className="flex grow flex-col gap-3">
-              {(item, items) => {
-                if (item.rounds_type === "AUTOMATED") {
+    <>
+      <div className="flex flex-1 justify-between">
+        <PageTitle title="Update Log" hideBack breadcrumbs={false} />
+        <DailyRoundsFilter setFilter={setFilter} />
+      </div>
+
+      <PaginatedList
+        route={routes.getDailyReports}
+        pathParams={{
+          consultationId,
+        }}
+        query={{
+          rounds_type: filter.roundType,
+        }}
```

Comment:
Rename the state variables to be same as the query param names. It's much easier to manage this group of states that way!

```suggestion
        query={filter}
```